app/build.gradle

```diff
@@ -51,18 +51,19 @@ android {
 }
 
 dependencies {
-    implementation group: 'com.android.support', name: 'support-v4', version: '27.1.0'
-    implementation group: 'com.android.support', name: 'design', version: '27.1.0'
-    implementation group: 'com.android.support', name: 'appcompat-v7', version: '27.1.0'
-    implementation group: 'com.android.support', name: 'recyclerview-v7', version: '27.1.0'
-    implementation group: 'com.android.support', name: 'cardview-v7', version: '27.1.0'
+    implementation group: 'com.android.support', name: 'support-v4', version: '27.1.1'
+    implementation group: 'com.android.support', name: 'design', version: '27.1.1'
+    implementation group: 'com.android.support', name: 'appcompat-v7', version: '27.1.1'
+    implementation group: 'com.android.support', name: 'recyclerview-v7', version: '27.1.1'
+    implementation group: 'com.android.support', name: 'cardview-v7', version: '27.1.1'
     implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.2'
 
-    implementation group: 'com.google.android.gms', name: 'play-services-gcm', version: '12.0.0'
-    implementation group: 'com.google.firebase', name: 'firebase-messaging', version: '12.0.0'
+    implementation group: 'com.google.android.gms', name: 'play-services-gcm', version: '12.0.1'
+    implementation group: 'com.google.firebase', name: 'firebase-messaging', version: '12.0.1'
 
     implementation group: 'com.zopim.android', name: 'sdk', version: '1.3.6.1'
     implementation group: 'com.zendesk', name: 'support', version: '2.0.0'
+    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
It looks like we're not actually using ConstraintLayout anywhere (Cmd + F 'constraint' only shows us this line). Can you remove the dependency if it's not being used?